get_hashsets.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AppConstants = require("./app-constants").init();
+
+const fs = require("fs");
+const path = require("path");
+const request = require("request");
+
+const pkg = require("./package.json");
+
+const HIBP_AUTH = 'Bearer ' + new Buffer(AppConstants.HIBP_API_TOKEN);
+
+
+function getBreachHashset(breach) {
+  if (breach.IsActive && breach.IsVerified && breach.DataClasses.includes("Email addresses")) {
+    console.log("Active, verified breach with email addresses: " + breach.Name);
+    const url = AppConstants.HIBP_API_ROOT + '/enterprisesubscriber/hashset/' + breach.Name;
+    console.log("url: " + url);
+
+    const headers = {
+      'User-Agent': `${pkg.name}@${pkg.version}`,
+      'Authorization': HIBP_AUTH
+    }
+    const hashsetRequestObject = {
+      url: url,
+      headers: headers
+    };
+
+    request(hashsetRequestObject).pipe(fs.createWriteStream('breach_hashsets' + path.sep + breach.Name + '.zip'));
+  }
+}
+
+function handleBreachesResponse(error, response, body) {
+  let breachesJSON = JSON.parse(body);
+  for (let breach of breachesJSON) {
+    getBreachHashset(breach);
+  }
+}
+
+const breachesRequestObject = {
+  url: AppConstants.HIBP_API_ROOT + '/breaches',
+  headers: {
+    'User-Agent': 'request'
+  }
+}
+
+request(breachesRequestObject, handleBreachesResponse);
```

Comment:
Actually, I'm not 100% sure how I'd use this file (especially since I'm not privileged enough to have an `HIBP_API_TOKEN`). If we're just calling this file directly from the CLI (a la `$ node get_hashsets`), we should document that in the README.md file, and/or optionally make it get called automatically via `npm start` or some other method so we know the expected hash sets are there before starting the server.